### ISSUE TITLE
Errors not displayed in named forms

### ISSUE THREAD

**Issue by dirkpostma** [kristijanhusak/laravel-form-builder#222]
I'd like to render multiple forms on a page, form A and form B.  They both have a field named 'somefield' field, that has some validation rules. When I submit form A and validation failed, the error is shown in both form A and B. Of course, I only want this error to be displayed in form A.

I read about the option of named forms, http://kristijanhusak.github.io/laravel-form-builder/form/named-form.html, I guess that is meant exactly for my usecase. However, when I set a name on a form, either by passing 'name' as option or by setting the property $name in de form class, I don't see errors at all when validation fails.

Questions:
1. Is using named forms the way to go for my use case?
2. If so: how should I get it working?

**Comment by kristijanhusak** [comment]
Yes, named forms should be the way to go, but it's really strange that errors are not shown. I'll check it out, thanks for reporting.

**Comment by terrasoff** [comment]
[request errors](https://github.com/kristijanhusak/laravel-form-builder/blob/master/src/views/errors.php) know nothing about your form names. Just [flat array of errors](https://github.com/laravel/framework/blob/5.2/src/Illuminate/Foundation/Http/FormRequest.php#L98)

**Comment by terrasoff** [comment]
[Errors](https://github.com/kristijanhusak/laravel-form-builder/blob/master/src/views/errors.php) come from response are just [flat array](https://github.com/laravel/framework/blob/5.2/src/Illuminate/Foundation/Http/FormRequest.php#L98). No mentions about form name :( 
But actually 'Validator' knows form name. Not sure if *Laravel* is able work with named forms correctly.

**Comment by wongwangki** [comment]
Try to use inline rules instead of $form->valiadate()

ie:
use: ```->add( 'title', 'text', [ 'rules' => 'min:3|max:255|required' ] )```
